test/integration/etcd_storage_path_test.go

```diff
@@ -989,7 +1001,7 @@ func TestEtcd3StoragePath(t *testing.T) {
 
 	_, err = testserver.StartConfiguredMasterAPI(masterConfig)
 	if err != nil {
-		t.Fatalf("error starting server: %#v", err)
+		t.Fatalf("error starting server: %v", err.Error())
```

Comment:
why .Error()?

Reply:
While we're nitting `t.Fatal(err)`.  It already gives you a line number.

Reply:
> why .Error()?

it was printing a pointer reference before, and needed the actual error text to figure out `--runtime-config=apis/...` was what was crashing this test. %v + err would probably suffice, but I didn't feel like revisiting

Reply:
nit, nvmd